src/js/components/Widgets/ReadMore.jsx

```diff
@@ -34,7 +34,7 @@ export default class ReadMore extends Component {
           link_text = "More";
         }
         if (collapse_text === undefined) {
-          collapse_text = " ...Less";
+          collapse_text = " ...Show Less  ";
```

Comment:
Okay - I exposed the (view source) link on mobile as part of this but made sure that the Show Less link stays inline with the text while the view source link wraps underneath. I'm fine with the Show Less link if it's useful but I want to be mindful of having lots of tiny little links near each other)